app/views/draws/show.html.erb

```diff
@@ -12,5 +12,8 @@
     </tbody>
   </table>
 </div>
+<% if can_start_draw?(draw: @draw, policy: policy(@draw)) %>
```

Comment:
`policy(@draw).start?` should work given my suggested change to the policy above